test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -165,28 +165,31 @@ def submit_job(self, sleep=10, lower=0,
         :param job_msg : Expected message upon submission failure
         :type  job_msg : int
 
+        :param running : Checks Job status R
+        :type  running : boolean(True/False)
+
         """
         arr_flag = False
         j = Job(TEST_USER)
         if((lower >= 0) and (upper > lower)):
             j.set_attributes({ATTR_J: '%d-%d' % (lower, upper)})
             arr_flag = True
-            total_jobs = upper - lower + 1
+            arr_jobs = {'total_jobs': '%d' % (upper - lower + 1)}
         j.set_sleep_time(sleep)
         try:
             jid = self.server.submit(j)
             if job_id is not None:
                 self.assertEqual(jid.split('.')[0], job_id)
             if arr_flag:
-                self.server.expect(JOB, {'job_state': 'B'}, id=jid)
-                self.server.expect(JOB, {'job_state=R': '%d' % (total_jobs)},
-                                   count=True, id=jid, extend='t')
-                if sleep == 1:
-                    self.server.expect(JOB, 'queue', op=UNSET, id=jid)
+                if running:
+                    self.server.expect(JOB, {'job_state': 'B'}, id=jid)
+                    self.server.expect(
+                        JOB,
+                        {'job_state=R': arr_jobs['total_jobs']},
+                        count=True, id=jid, extend='t')
             else:
-                self.server.expect(JOB, {'job_state': 'R'}, id=jid)
-                if sleep == 1:
```

Comment:
not sure what the sleep == 1 check was for, but can you please comment on why you removed it?